src/lib/Pagination/Pagerfanta/ReverseRelationAdapter.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Pagination\Pagerfanta;
+
+use eZ\Publish\API\Repository\ContentService;
+use eZ\Publish\API\Repository\Values\Content\Content;
+use EzSystems\EzPlatformAdminUi\UI\Dataset\DatasetFactory;
+use Pagerfanta\Adapter\AdapterInterface;
+
+class ReverseRelationAdapter implements AdapterInterface
+{
+    /** @var \eZ\Publish\API\Repository\ContentService */
+    private $contentService;
+
+    /** @var \EzSystems\EzPlatformAdminUi\UI\Dataset\DatasetFactory */
+    private $datasetFactory;
+
+    /** @var \eZ\Publish\API\Repository\Values\Content\Content */
+    private $content;
+
+    /**
+     * @param \eZ\Publish\API\Repository\ContentService $contentService
+     * @param \EzSystems\EzPlatformAdminUi\UI\Dataset\DatasetFactory $datasetFactory
+     * @param \eZ\Publish\API\Repository\Values\Content\Content $content
+     */
+    public function __construct(
+        ContentService $contentService,
+        DatasetFactory $datasetFactory,
+        Content $content
+    ) {
+        $this->contentService = $contentService;
+        $this->datasetFactory = $datasetFactory;
+        $this->content = $content;
+    }
+
+    /**
+     * Returns the number of results.
+     *
+     * @return int the number of results
+     */
+    public function getNbResults()
```

Comment:
I do not like name od the function, `count()` maybe ?

Reply:
afaik both method names comes from `Pagerfanta\Adapter\AdapterInterface`

Reply:
ok